cli/config/configfile/file.go

```diff
@@ -36,6 +36,10 @@ type ConfigFile struct {
 	Filename             string                      `json:"-"` // Note: for internal use only
 	ServiceInspectFormat string                      `json:"serviceInspectFormat,omitempty"`
 	ServicesFormat       string                      `json:"servicesFormat,omitempty"`
+	HTTPProxy            string                      `json:"httpProxy,omitempty"`
+	HTTPSProxy           string                      `json:"httpsProxy,omitempty"`
+	NoProxy              string                      `json:"noProxy,omitempty"`
+	FTPProxy             string                      `json:"ftpProxy,omitempty"`
```

Comment:
it would be really nice if the build args were nested under a "BuildArgs" property - then we wouldn't need to prefix them.  But I'm not sure if our config file supports nest things.